packages/pluto/src/__module.d.ts

```diff
@@ -0,0 +1,3 @@
+// module declarations
+declare module 'md5';
```

Comment:
Why do we need both? and I couldn't find them on package.json.

Reply:
We don't, this seems to be left over from the package split.